pennylane/drawer/draw.py

```diff
@@ -24,6 +24,29 @@
 from .tape_mpl import tape_mpl
 from .tape_text import tape_text
 
+_level_sentinel = object()

```

Comment:
I don't understand why we can't use a value here that we _know_ to be invalid and is very unlikely to be used by a user? Like -1, or even something like `INTEGER_MIN_VAL` to be even more sure.

Reply:
Integer sentinels can not be used since the underlying `get_transform_program` function accepts integers to build a `slice` object for selecting the wanted transforms. So `-1` may result in something like `slice(0, -1)`. `INTEGER_MIN_VAL` is definitely a safer alternative but device-dependent, right? Also it sounds like a waste of memory, so something much more simplistic like `object()` feels sufficient.

Reply:
> Also it sounds like a waste of memory

It's going to take up just as much memory as an integer (4 bytes). You can get it with `np.iinfo(np.int32).min`

Reply:
Also, sorry to be nitpicky but for the sake of simplicity I think a different name might be better. Personally I had to look up what a sentinel.

Reply:
Also note that this part is temporary stopgap till we can get rid of the `expansion_strategy` keywarg in favor of only `level`.

Reply:
Ok in that case it doesn't really matter. 😅